BSB_LAN/BSB_LAN.ino

```diff
@@ -528,7 +528,7 @@ EEPROMClass EEPROM_ESP("nvs", EEPROM_SIZE);
 
 #include <Wire.h>
 #include "src/BlueDot_BME280/BlueDot_BME280.h"
-BlueDot_BME280 bme[BME280];  //Set 2 if you need two sensors.
+BlueDot_BME280 *bme;  //Set 2 if you need two sensors.
```

Comment:
How do we set to "2" in this line now?

Reply:
in setup():
bme = new BlueDot_BME280[BME_Sensors];

Reply:
Ah, of course, stupid me ;)...

Reply:
But then the comment "Set to 2" should be moved there, or into _config.h?